<div class="col-8 offset-2">
    <h1>Functionally known homologs (as of May 2018)</h1>

    <p>&nbsp;</p>

    <h3>by S. Frillingos</h3>

    <h3>Laboratory of Biological Chemistry, Department of Medicine, University of Ioannina, Greece</h3>

    <p>&nbsp;</p>

    <h2>Subfamily SF1 Cluster C1, synonyms: XanQ/UapA cluster</h2>

    <h2>(xanthine/uric acid transporter cluster)</h2>

    <p>&nbsp;</p>

    <h3>Functionally known prokaryotic homologs</h3>

    <p>&nbsp;</p>

    <h4>XanQ of Escherichia coli K-12: Xanthine permease</h4>

    <h4>UniProt_ P67444</h4>

    <p>The <em>xanQ </em>(<em>ygfO</em>) gene is part of a gene cluster including genes for purine catabolism (such as several subunits of xanthine dehydrogenase) and 2 other NAT/NCS2 genes for putative purine transporters (<em>ghxQ, uacT</em>) (Xi et al., 2000);<em> xanQ</em> (<em>ygfO</em>) and other genes of this cluster show enhanced transcript levels at nitrogen-limited conditions associated with regulation by NtrC/Nac (Zimmer et al., 2000)</p>

    <p>XanQ (YgfO) is a high-affinity xanthine permease; it is specific for xanthine (<em>K</em><sub>m</sub> = 4.5 μΜ) and does not recognize uric acid or 7-methyl- or 8-methylxanthine (Karatza and Frillingos, 2005); oxypurinol (Karena et al., 2015), 1-methyl-, 3-methyl-, 9-methyl-, 2-thio- or 6-thioxanthine and 8-thiouric acid (Karatza et al., 2006) are recognized with moderate affinities (<em>K</em><sub>i</sub> 10- to 25-fold higher than the <em>K</em><sub>m</sub> for xanthine); xanthine uptake by XanQ is proton-gradient dependent (Karatza and Frillingos, 2005)</p>

    <p>The specificity of XanQ for xanthine is linked with presence of Asn93 at the binding site (Karena and Frillingos, 2009; Karena and Frillingos, 2011; Karena et al., 2015)</p>

    <p>&nbsp;</p>

    <h4>XanP of Escherichia coli K-12: Xanthine permease</h4>

    <h4>UniProt_ P0AGM9</h4>

    <p>XanP (YicE) is a high-affinity xanthine permease; it is specific for xanthine (<em>K</em><sub>m</sub> = 3.5 μΜ) and does not recognize uric acid or 7-methyl- or 8-methylxanthine (Karatza and Frillingos, 2005); xanthine uptake by XanP is proton-gradient dependent (Karatza and Frillingos, 2005); amplified expression of plasmid-borne <em>xanP</em> through <em>lacZ</em>p/o resulted in low protein levels of XanP in the <em>E. coli</em> membrane fraction (Karatza and Frillingos, 2005)</p>

    <p>The <em>xanP (yicE)</em> gene belongs to the purine-responsive PurR regulon of <em>E. coli</em> and is repressed by PurR in response to adenine (Cho et al., 2011)</p>

    <p>&nbsp;</p>

    <h4>UacT of Escherichia coli K-12: Uric acid permease</h4>

    <h4>UniProt_ Q46821</h4>

    <p>The <em>uacT </em>(<em>ygfU</em>) gene is part of a gene cluster including genes for purine catabolism (such as several subunits of xanthine dehydrogenase) and 2 other NAT/NCS2 genes for putative purine transporters (<em>xanQ</em>, <em>ghxQ</em>) (Xi et al., 2000)</p>

    <p>UacT (YgfU) is a uric acid permease; it transports uric acid with rather low affinity (<em>K</em><sub>m</sub> 0.5 mM); it also transports xanthine but with disproportionally low efficiency and only at 37° C (Papakostas and Frillingos, 2012); the preference for uric acid over xanthine is associated with presence of Thr100 at the binding site (Papakostas and Frillingos, 2012; Frillingos, 2012); uric acid uptake by UacT is proton-gradient dependent (Papakostas and Frillingos, 2012)</p>

    <p>&nbsp;</p>

    <h4>AcS4X6 of Acinetobacter calcoaceticus: probable xanthine transporter</h4>

    <h4>UniProt_ DOS4X6</h4>

    <p>It corresponds to part of a clone from a gut metagenomic library allowing xanthine-dependent growth and xanthine uptake in a synthetic biosensor-based assay (Genee et al., 2016)</p>

    <p>&nbsp;</p>

    <h4>OXL43103 of Prevotella copri: probable xanthine transporter</h4>

    <h4>UniProt_ A0A229I2P1</h4>

    <p>It corresponds to part of a clone from a gut metagenomic library allowing xanthine-dependent growth in a synthetic biosensor-based assay (Genee et al., 2016)</p>

    <p>&nbsp;</p>

    <h4>PbuX of Bacillus subtilis: xanthine permease</h4>

    <h4>UniProt_ P42086</h4>

    <p>The <em>pbuX</em> gene belongs to the <em>xpt-pbuX</em> operon which is repressed by hypoxanthine and guanine; <em>xpt</em> encodes a specific xanthine phosphoribosyltransferase (XPRTase); disruption of <em>pbuX</em> leads to abolishment of xanthine uptake activity but retention of the XPRTase activity; the <em>pbuX</em>-dependent xanthine uptake activity of <em>B. subtilis</em> has a <em>K</em><sub>m</sub> of 1 μΜ (Christiansen et al., 1997); the <em>xpt-pbuX</em> operon belongs to the PurR regulon (Saxild et al., 2001)</p>

    <p>The <em>xpt-pbuX</em> operon is repressed by hypoxanthine and guanine at the RNA level through a guanine-responsive riboswitch (Johansen et al., 2003; Mandal et al., 2003; Batey et al., 2004); at high ionic strength (20 mM MgCl<sub>2</sub>) the <em>xpt-pbuX</em> RNA has high affinity for guanine (<em>K</em><sub>d</sub> 5 nM) followed by xanthine and hypoxanthine (<em>K</em><sub>d</sub> 50-70 nM) (Mandal et al., 2003); at more physiological ionic strength (0.25 to 1 mM MgCl<sub>2</sub>), the affinity of <em>xpt-pbuX</em> RNA for hypoxanthine (<em>K</em><sub>d</sub> 3-4 μΜ) is similar to the affinity for hypoxanthine of the <em>E. coli</em> PurR repressor protein (<em>K</em><sub>d</sub> 9 μΜ) (Batey et al., 2004)</p>

    <p>&nbsp;</p>

    <h4>PucJ of Bacillus subtilis: probable uric acid permease</h4>

    <h4>UniProt_ O32139</h4>

    <p>The <em>pucJ</em> gene belongs to a cluster of 14 genes linked with purine catabolism that are controlled by the PucR transcription activator and, with one exception, are adjacent to each other in the genome of <em>B. subtilis</em> (Schultz et al., 2001); <em>pucJ</em> (<em>yunJ</em>) and <em>pucK </em>(<em>yunK</em>) were proposed to encode a uric acid transport system; PucJ and PucK are closely related in sequence to PbuX; disruption of <em>pucJ </em>or <em>pucK </em>disrupts the ability of <em>B. subtilis</em> to grow on 1 mM uric acid (Schultz et al., 2001)</p>

    <p>&nbsp;</p>

    <h4>PucK of Bacillus subtilis: probable uric acid permease</h4>

    <h4>UniProt_ O32140</h4>

    <p>The <em>pucK</em> gene belongs to a cluster of 14 genes linked with purine catabolism that are controlled by the PucR transcription activator and, with one exception, are adjacent to each other in the genome of <em>B. subtilis</em> (Schultz et al., 2001); <em>pucJ</em> (<em>yunJ</em>) and <em>pucK </em>(<em>yunK</em>) were proposed to encode a uric acid transport system; PucJ and PucK are closely related in sequence to PbuX; disruption of <em>pucJ </em>or <em>pucK </em>disrupts the ability of <em>B. subtilis</em> to grow on 1 mM uric acid (Schultz et al., 2001)</p>

    <p>&nbsp;</p>

    <h3>Functionally known eukaryotic homologs</h3>

    <p>&nbsp;</p>

    <h4>UapA of Aspergillus nidulans: xanthine/uric acid permease</h4>

    <h4>UniProt_ Q07307</h4>

    <p>Early genetic studies with mutants resistant to various purine analogs had indicated that <em>Aspergillus nidulans</em> contains two major transport systems for uptake of purine bases, the one for uric acid and xanthine (identified later as UapA) and the other for hypoxanthine, adenine and guanine (identified later as AzgA); the UapA system had been hypothesized on the basis of the isolation of mutants resistant to (the yellow-conidial effect of) 2-thio<u>u</u>ric <u>a</u>cid (<em>uap </em>mutants) (Darlington and Scazzocchio, 1967); the <em>uapA</em> gene encodes a major uric acid-xanthine permease of <em>A. nidulans</em> (Diallinas and Scazzocchio, 1989); <em>uapA</em> transcription is inducible by 2-thiouric acid (Diallinas and Scazzocchio, 1989) and repressible by ammonium (Gorfinkiel et al., 1993)</p>

    <p>UpaA is a high-affinity permease for both uric acid (<em>K</em><sub>m</sub> 7 μΜ) (Diallinas et al., 1998) and xanthine (<em>K</em><sub>m</sub> 8 μΜ) (Koukaki et al., 2005); it does not bind or transport guanine, hypoxanthine, adenine, or uracil, but recognizes with high/moderate affinity several xanthine analogs: 2-thio-, 3-methyl-, 8-methylxanthine, oxypurinol (<em>K</em><sub>i</sub> 0.03-0.1 mΜ, 4-fold to 15-fold higher than the <em>K</em><sub>i</sub> for uric acid, as measured in xanthine-transport competition assays), 1-methyl-, 6-thio-, 8-aza-, 9-methylxanthine, 2-, 6- or 8-thiouric acid (<em>K</em><sub>i</sub> 0.2-0.5 mM; 30- to 70-fold higher than the <em>K</em><sub>i</sub> for uric acid) (Koukaki et al., 2005), 8-bromoxanthine (<em>K</em><sub>i</sub> 0.65 mM) (Alguel et al., 2016)</p>

    <p>Based on <em>in vivo</em> complementation between sequences of UapA and of UapC, a highly homologous paralog of UapA that is a broader-specificity permease, and analysis of engineered UapA/UapC chimeras, specificity-related changes were found to localize in a conserved motif (Diallinas et al., 1998) that was later termed NAT (Nucleobase-Ascorbate Transporter) signature motif; functionally essential residues of this motif were identified in both UapA (Koukaki et al., 2005) and the related homolog XanQ of <em>Escherichia coli</em> (Karatza et al., 2006); the NAT motif is an 11-residue motif at TM10 of NAT/NCS2 transporters including a conserved substrate-binding residue Gln/Glu (Gln408 in UapA) (Alguel et al., 2016); genetic screens in UapA identified amino acid changes distant from the xanthine/uric-acid binding site that seem to broaden the substrate profile (allowing utilization of hypoxanthine or adenine) without affecting the affinity/kinetics for xanthine or uric acid; these changes may reflect substrate-selectivity gates that control access of substrates to the binding site; the observed effects are more prominent in mutants combining changes at “selectivity” gates with changes at the binding site (Diallinas, 2014)</p>

    <p>Dimeric crystal structure of a genetically stabilized version of UapA (G411V<sub>Δ1-11</sub>) bound with xanthine in an inward-open conformation at 3.7 Å (PDB ID 5I6C); UapA dimerization may play a role in membrane trafficking and transport activity of UapA (Alguel et al., 2016)</p>

    <p>&nbsp;</p>

    <h4>UapC of Aspergillus nidulans: broad-profile xanthine/uric acid permease</h4>

    <h4>UniProt_ P48777</h4>

    <p>Loss-of-function mutations of <em>uapA</em> or <em>azgA</em> in <em>Aspergillus nidulans</em>, encoding the major uric-acid xanthine or hypoxanthine-adenine-guanine permease, respectively, result in impaired utilization of these purines as sole nitrogen sources; the residual growth is due to the activity of a third, broad-specificity purine permease, encoded by the <em>uapC</em> gene (Diallinas et al., 1995); UapC is closely related in sequence with UapA and is a uric acid-xanthine permease which also recognizes hypoxanthine and other purines (Diallinas et al., 1998); the xanthine-uptake activity of UapC is proton-gradient dependent; UapC displays high affinity for xanthine (<em>K</em><sub>m</sub> 3.3 μM) but lower affinity for uric acid (<em>K</em><sub>i</sub> 135 μΜ) and recognizes several other purines and purine analogs in transport-competition assays (Krypotou and Diallinas, 2014)</p>

    <p>A chimeric UapA/UapC transporter (Diallinas et al., 1998; Papageorgiou et al., 2008) consisting of UapA having the sequence 344-436 (TM8-TM11) grafted from UapC displays the same kinetics properties as UapC (<em>K</em><sub>m</sub> 2.8 μM for xanthine and <em>K</em><sub>i</sub> 130 μΜ for uric acid, as opposed to UapA which shows comparably high-affinity for both uric acid and xanthine), consistent with the notion that TM8-TM11 contains all the major substrate-binding elements (Krypotou and Diallinas, 2014)</p>

    <p>&nbsp;</p>

    <h4>AfUapC of Aspergillus fumigatus: xanthine/uric acid permease</h4>

    <h4>UniProt_ Q4WGR0</h4>

    <p>Proton-gradient dependent xanthine/uric-acid transporter with properties similar to UapC of <em>Aspergillus nidulans</em>; it displays high affinity for xanthine (<em>K</em><sub>m</sub> 1.5 μM in <em>A. fumigatus</em>, or 5.5 μM when heterologously expressed in <em>A. nidulans</em>), lower affinity for uric acid (<em>K</em><sub>i</sub> 12.5 μM in <em>A. fumigatus</em>, or 89 μM when expressed in <em>A. nidulans</em>), oxypurinol (<em>K</em><sub>i</sub> 75 μM in <em>A. fumigatus</em>, or 103 μM when expressed in <em>A. nidulans</em>) and several purine analogs (Goudela et al., 2008)</p>

    <p>&nbsp;</p>

    <h4>Xut1p of Candida albicans: xanthine/uric acid permease</h4>

    <h4>UniProt_ Q5A1D7</h4>

    <p>Proton-gradient dependent xanthine/uric acid transporter, with high affinity for xanthine (<em>K</em><sub>m</sub> 4 μM) and oxypurinol (<em>K</em><sub>i</sub> 4 μM), lower affinity for uric acid (<em>K</em><sub>i</sub> 50 μM) and several xanthine/uric acid analogs; it does not bind hypoxanthine or guanine (Goudela et al., 2005)</p>

    <p>&nbsp;</p>

    <h4>AbUapC of Aspergillus brasiliensis: broad-profile xanthine/uric acid permease</h4>

    <p>High-capacity, high-affinity xanthine/uric acid transporter with characteristics similar to UapC of <em>Aspergillus nidulans</em>; AbUApC displays relatively high affinity for xanthine (<em>K</em><sub>m</sub> 23 μM) but lower affinity for uric acid (<em>K</em><sub>i</sub> 160 μΜ) and recognizes also hypoxanthine (<em>K</em><sub>i</sub> 0.7 mΜ) and several purine analogs in transport-competition assays (Krypotou et al., 2016)</p>

    <p>&nbsp;</p>

    <h4>AbUapD of Aspergillus brasiliensis: no apparent transporter function</h4>

    <p>AbUapD represents a new phylogenetic clade of dikaryotic fungal NAT/NCS2s that is distinct from the ones of UapC and UapA; UapD sequences lack the two essential substrate-binding residues that are conserved in all functional NAT/NCS2s (a Glu in TM8 and a Glu/Gln in TM10, which are replaced by Ser/Leu/Val and Gly/Ala, respectively, in UapDs) (Krypotou et al., 2016); heterologous expression of AbUapD (as well as of several AbUapD/UapA chimeras) in <em>Aspergillus nidulans</em> resulted in ER-retention and vacuolar degradation, suggesting that AbUapD may represent an intrinsically misfolded protein, corresponding possibly to a proto-pseudogene or proto-neogene (Krypotou et al., 2016)</p>

    <p>&nbsp;</p>

    <h4>PhU of Phanerochaete chrysosporium: xanthine and uric acid transporter</h4>

    <p>PhU is a xanthine and uric acid transporter with properties similar to UapA (based on growth tests on purines and other nitrogen sources and protection against the cytotoxicity of oxypurinol in growth tests combining oxypurinol with other purines or pyrimidines as putative competitors) (Barrago-Vega et al., 2017)</p>

    <p>&nbsp;</p>

    <h2>Subfamily SF1 Cluster C2, synonyms: UraA cluster (uracil transporter cluster)</h2>

    <p>&nbsp;</p>

    <h3>Functionally known prokaryotic homologs</h3>

    <p>&nbsp;</p>

    <h4>UraA of Escherichia coli K-12: Uracil permease</h4>

    <h4>UniProt_P0AGM7</h4>

    <p>UraA is a uracil permease; disruption of the <em>uraA</em> locus confers resistance to 5-FU (5-fluorouracil) and abolishes uracil transport (Andersen et al., 1995)</p>

    <p>UraA binds and transports uracil with high affinity (<em>K</em><sub>d</sub> = 0.4 μΜ; <em>K</em><sub>m</sub> = 0.5 μΜ); Monomeric crystal structure of UraA bound to uracil in inward open conformation at 2.8 Å (PDB ID 3QE7) (Lu et al., 2011)</p>

    <p>Dimeric crystal structure of UraA bound to uracil in occluded conformation at 2.5 Å (PDB ID 5XLS); Dimer formation is required for the transport activity of UraA (Yu et al., 2017)</p>

    <p>UraA is specific for uracil (<em>K</em><sub>m</sub> = 0.2 μΜ) and 5-FU (<em>K</em><sub>i</sub> = 1.8 μΜ); steric hindrance with Phe73 excludes thymine from the binding site; the uracil uptake activity is proton-gradient dependent (Botou et al., 2018)</p>

    <p>&nbsp;</p>

    <h4>RutG of Escherichia coli K-12: Uracil/thymine permease with broad specificity</h4>

    <h4>UniProt_P75892</h4>

    <p>The <em>rutG</em> gene belongs to the<em> rut</em> (py<u>r</u>imidine <u>ut</u>ilization) operon; it is sporadically present in enterobacteria and in few other proteobacteria outside Enterobacteriales (Loh et al., 2006)</p>

    <p>Disruption of <em>rutG </em>leads to poor growth on uracil or thymine (Kim et al., 2010)</p>

    <p>RutG is a high-affinity permease for uracil (<em>K</em><sub>m</sub> = 0.5 μΜ) and thymine (<em>K</em><sub>m</sub> = 0.3 μΜ), it also transports xanthine (<em>K</em><sub>m</sub> = 65 μΜ), and is inhibited by 5-FU (<em>K</em><sub>i</sub> = 0.7 μΜ) and oxypurinol (<em>K</em><sub>i</sub> = 35 μΜ); the uracil uptake activity is proton-gradient dependent (Botou et al., 2018)</p>

    <p>&nbsp;</p>

    <h4>AvDDG3 of Aeromonas veronii: Uracil permease</h4>

    <h4>UniProt_F4DDG3</h4>

    <p>AvDDG3 is a high-affinity permease for uracil (<em>K</em><sub>m</sub> = 0.3 μΜ) and 5-FU (<em>K</em><sub>i</sub> = 2.0 μΜ); the uracil uptake activity is proton-gradient dependent (Botou et al., 2018)</p>

    <p>&nbsp;</p>

    <h4>AcS572 of Acinetobacter calcoaceticus: Uracil permease</h4>

    <h4>UniProt_DOS572</h4>

    <p>AcS72 is a high-affinity permease for uracil (<em>K</em><sub>m</sub> = 0.8 μΜ) and 5-FU (<em>K</em><sub>i</sub> = 2.9 μΜ); the uracil uptake activity is proton-gradient dependent (Botou et al., 2018)</p>

    <p>&nbsp;</p>

    <h4>PyrP of Bacillus subtilis: probable uracil permease</h4>

    <p>The <em>pyrP</em> gene belongs to a <u>pyr</u>imidine biosynthetic (<em>pyr</em>) gene cluster of <em>Bacillus subtilis</em>; PyrP was proposed to be a uracil permease (Turner et al., 1994)</p>

    <p>&nbsp;</p>

    <h4>PyrP of Lactobacillus plantarum: probable uracil permease</h4>

    <p>Disruption of <em>pyrP</em> confers increased resistance to 5-FU and abolishes uracil uptake (Arsène-Ploetze et al., 2006)</p>

    <p>&nbsp;</p>

    <h2>Subfamily SF2, synonyms: AzgA-like, COG2252, AdeP/GhxP cluster</h2>

    <h2>(adenine/guanine/hypoxanthine transporter cluster)</h2>

    <p>&nbsp;</p>

    <h3>Functionally known prokaryotic homologs</h3>

    <p>&nbsp;</p>

    <h4>AdeP of Escherichia coli K-12: Adenine permease</h4>

    <h4>UniProt_P31466</h4>

    <p>The <em>adeP </em>gene product was annotated initially as an adenine transporter and given the name <em>purP</em> (<u>pur</u>ine <u>p</u>ermease) (original designation was <em>yieG</em>) based on genetic studies with <em>E. coli</em> mutants that were defective in utilization of adenine (Burton, 1983; Burton, 1984)</p>

    <p>The <em>adeP (purP)</em> gene belongs to the PurR regulon of <em>E. coli</em> and is directly repressed by PurR in response to adenine (Cho et al., 2011)</p>

    <p>AdeP (PurP) is a high-affinity adenine permease; it transports adenine (<em>K</em><sub>m</sub> = 1.0 μΜ) but not guanine, hypoxanthine, xanthine, uric acid or uracil; it recognizes with high affinity (<em>K</em><sub>i</sub> &lt; 10 μΜ) 2,6-diaminopurine and purine, and with lower affinity (<em>K</em><sub>i</sub> 0.1 mΜ) 9-methyladenine; hypoxanthine is recognized with low affinity (<em>K</em><sub>i</sub> 0.3 mM); guanine, 6-thioguanine or 6-mercaptopurine are not recognized significantly (Papakostas et al., 2013); the endogenous <em>adeP</em> (<em>purP</em>) gene is solely responsible for the uptake of adenine in aerobically grown <em>E. coli </em>(Papakostas et al., 2013)</p>

    <p>&nbsp;</p>

    <h4>GhxP of Escherichia coli K-12: Guanine/hypoxanthine permease</h4>

    <h4>UniProt_P0AF52</h4>

    <p>Initial genetic studies indicated that <em>E. coli</em> uses a transport system for the uptake of guanine and hypoxanthine that is distinct from the adenine transport system (Roy-Burman et al., 1975)</p>

    <p>The <em>ghxP (yjcD)</em> gene belongs to the PurR regulon of <em>E. coli</em> (Marbach et al., 2012)</p>

    <p>Disruption of the <em>ghxP</em> (<em>yjcD</em>) gene (but not of any other NAT/NCS2 or NCS1 gene) reduces the <em>E. coli</em> sensitivity to cytotoxic analogs 6-N-hydroxylaminopurine (HAP), 2-amino-HAP, 6-mercaptopurine, 6-thioguanine, 2-aminopurine; GhxP (YjcD) might be a principal permease for guanine and hypoxanthine (Kozmin et al., 2013)</p>

    <p>GhxP (YjcD) is a high-affinity guanine and hypoxanthine permease; it transports guanine (<em>K</em><sub>m</sub> = 1.6 μΜ) and hypoxanthine (<em>K</em><sub>m</sub> = 11.2 μΜ) but not adenine, xanthine, uric acid or uracil; it recognizes with high affinity 6-thioguanine (<em>K</em><sub>i</sub> = 3.5 μΜ) and 6-mercaptopurine (<em>K</em><sub>i</sub> = 16.5 μΜ), with lower affinity 1-methylguanine and 8-azaguanine (<em>K</em><sub>i</sub> 30-40 μΜ); adenine, 2,6-diaminopurine, or other adenine-related analogs are not recognized (Papakostas et al., 2013); the endogenous <em>ghxP</em> (<em>yjcD</em>) gene is solely responsible for the uptake of guanine and hypoxanthine in aerobically grown <em>E. coli </em>(Papakostas et al., 2013)</p>

    <p>&nbsp;</p>

    <h4>AdeQ of Escherichia coli K-12: Adenine permease</h4>

    <h4>UniProt_P31440</h4>

    <p><em>adeQ</em> (<em>yicO</em>) is a closely related paralog of <em>adeP</em>, but, in contrast to <em>adeP</em>, is present only sporadically in Enterobacteriales and is absent in several <em>E. coli </em>strains</p>

    <p>AdeQ (YicO) is a high-affinity adenine permease; it recognizes the same substrates and ligands as AdeP (with which it shares 73% identity in sequence) but with lower affinities relative to AdeP; AdeQ transports adenine (<em>K</em><sub>m</sub> = 6.5 μΜ) but not guanine, hypoxanthine, xanthine, uric acid or uracil; it recognizes with moderate affinity purine (<em>K</em><sub>i</sub> 20 μΜ) and 2,6-diaminopurine (<em>K</em><sub>i</sub> 70 μΜ); hypoxanthine is recognized with low affinity (<em>K</em><sub>i</sub> 0.3 mM); guanine, 6-thioguanine or 6-mercaptopurine (6-MP) are not recognized significantly (Papakostas et al., 2013)</p>

    <p>&nbsp;</p>

    <h4>GhxQ of Escherichia coli K-12: Guanine/hypoxanthine permease</h4>

    <h4>UniProt_Q46817</h4>

    <p><em>ghxQ</em> (<em>ygfQ</em>) is a closely related paralog of <em>ghxP (yjcD)</em></p>

    <p>The <em>ghxQ </em>(<em>ygfQ</em>) gene is part of a gene cluster including genes for purine catabolism (such as several subunits of xanthine dehydrogenase) and 2 other NAT/NCS2 genes for putative purine transporters (<em>xanQ, uacT</em>) (Xi et al., 2000);<em> ghxQ</em> (<em>ygfQ</em>) and other genes of this cluster show enhanced transcript levels at nitrogen-limited conditions associated with regulation by NtrC/Nac (Zimmer et al., 2000)</p>

    <p>GhxQ (YgfQ) is a high-affinity guanine and hypoxanthine permease; it recognizes the same substrates and ligands as GhxP (with which it shares 81% sequence identity) but with lower affinities for hypoxanthine, 6-mercaptopurine, 1-methylguanine and 8-azaguanine; it transports guanine (<em>K</em><sub>m</sub> = 1.8 μΜ) and hypoxanthine (<em>K</em><sub>m</sub> = 22.5 μΜ) but not adenine, xanthine, uric acid or uracil; it recognizes 6-thioguanine (<em>K</em><sub>i</sub> = 4 μΜ) and with lower affinity 6-mercaptopurine (<em>K</em><sub>i</sub> 55 μΜ), 1-methylguanine (<em>K</em><sub>i</sub> 0.1 mM), 8-azaguanine (<em>K</em><sub>i</sub> 0.15 mM); adenine, 2,6-diaminopurine, or other adenine-related analogs are not recognized (Papakostas et al., 2013)</p>

    <p>&nbsp;</p>

    <h4>BB_B22 of Borrelia burgdorferi: Adenine/guanine/hypoxanthine permease</h4>

    <h4>UniProt_O50993</h4>

    <p>The <em>bbb22</em> and <em>bbb23</em> genes are closely related in sequence and adjacent in the genome, in a critical virulence plasmid (cp26) of the Lyme-disease spirochete <em>Borrelia burgdorferi</em> (Jewett et al., 2007). The spirochete is dependent on purine salvage from the host environment for survival (Jewett et al., 2009). The <em>bbb22</em> and <em>bbb23</em> of <em>B. burgdorferi</em> B31 encode purine permeases (BB_B22, BB_B23) that are needed for <em>B. burgdorferi</em> mouse infectivity (Jain et al., 2012)</p>

    <p>BB_B22 and/or BB_B23 is essential for hypoxanthine transport and contributes to the transport of adenine and guanine, but not xanthine, cytosine, thymine or uracil (Jain et al., 2012)</p>

    <p>BB_B22 is a hypoxanthine and adenine transporter with apparent <em>K</em><sub>m</sub> 2.6-3.4 μΜ (hypoxanthine) and 0.13-0.19 μM (adenine) (Jain et al., 2015). Adenine uptake by BB_B22 is essential for <em>B. burgdorferi</em> survival in mice; hypoxanthine uptake by BB_B22 is also critical, but maximal hypoxanthine uptake requires the presence of BB_B23 as well (Jain et al., 2015)&nbsp; &nbsp;&nbsp;</p>

    <p>&nbsp;</p>

    <h4>BB_B23 of Borrelia burgdorferi: Adenine/guanine/hypoxanthine permease</h4>

    <h4>UniProt_O50994</h4>

    <p>The <em>bbb22</em> and <em>bbb23</em> genes are closely related in sequence and adjacent in the genome, in a critical virulence plasmid (cp26) of the Lyme-disease spirochete <em>Borrelia burgdorferi</em> (Jewett et al., 2007). The spirochete is dependent on purine salvage from the host environment for survival (Jewett et al., 2009). The <em>bbb22</em> and <em>bbb23</em> of <em>B. burgdorferi</em> B31 encode purine permeases (BB_B22, BB_B23) that are needed for <em>B. burgdorferi</em> mouse infectivity (Jain et al., 2012)</p>

    <p>BB_B22 and/or BB_B23 is essential for hypoxanthine transport and contributes to the transport of adenine and guanine, but not xanthine, cytosine, thymine or uracil (Jain et al., 2012)</p>

    <p>The <em>bbb23</em> gene is dispensable for <em>B. burgdorferi</em> mouse infectivity, but is required together with <em>bbb22</em> for maximal spirochete loads in infected mouse tissues (Jain et al., 2015).</p>

    <p>Spirochetes carrying <em>bbb22</em> alone are able to achieve wild-type levels of adenine saturation but not hypoxanthine saturation, suggesting that maximal hypoxanthine uptake requires the presence of <em>bbb23</em> (Jain et al., 2015)</p>

    <p>&nbsp;</p>

    <h4>PbuG of Bacillus subtilis: guanine/hypoxanthine permease</h4>

    <h4>UniProt_O34987</h4>

    <p>The <em>pbuG</em> gene (<u>p</u>urine <u>b</u>ase <u>u</u>ptake, 8-aza<u>g</u>uanine) belongs to the PurR regulon (Saxild et al., 2001); it is negatively regulated by hypoxanthine and guanine through a guanine riboswitch (Johansen et al., 2003; Mandal et al., 2003); disruption of the <em>pbuG</em> locus confers resistance to 0.5 mM 8-azaguanine (Saxild and Nygaard, 1987) and results in impaired hypoxanthine uptake (Saxild et al., 2001): PbuG was proposed to be a guanine/hypoxanthine permease (Saxild and Nygaard, 1987).</p>

    <p>&nbsp;</p>

    <h4>PbuO of Bacillus subtilis: probable guanine/hypoxanthine permease</h4>

    <h4>UniProt_O34978</h4>

    <p>The <em>pbuO</em> gene (<u>p</u>urine <u>b</u>ase <u>u</u>ptake, 6-<u>o</u>xopurine) belongs to the PurR regulon (Saxild et al., 2001); disruption of the <em>pbuO </em>locus confers resistance to 6-oxopurine and results in 50% reduction in guanine and hypoxanthine uptake activity (Saxild et al., 2001); the gene product is closely related in sequence with PbuG (47% identity): PbuO was proposed to be a guanine/hypoxanthine permease (Saxild et al., 2001).</p>

    <p>&nbsp;</p>

    <h4>PlAzg1 of Paenibacillus larvae: high-affinity adenine permease with broad specificity for purines/pyrimidines</h4>

    <p>High-affinity transporter for adenine (<em>K</em><sub>m</sub> 3 μΜ) with broad solute specificity profile, also found to transport hypoxanthine, xanthine, and uracil (Alexander et al., 2018)</p>

    <p>&nbsp;</p>

    <h4>PlAzg2 of Paenibacillus larvae: high-affinity adenine permease with broad specificity for purines/pyrimidines</h4>

    <p>High-affinity transporter for adenine (<em>K</em><sub>m</sub> 2 μΜ) with broad solute specificity profile, also found to transport guanine, hypoxanthine, xanthine, cytosine, uracil (Alexander et al., 2018)</p>

    <p>&nbsp;</p>

    <h3>Functionally known eukaryotic homologs</h3>

    <p>&nbsp;</p>

    <h4>AzgA of Aspergillus nidulans: Adenine/guanine/hypoxanthine permease</h4>

    <h4>UniProt_Q7Z8R3</h4>

    <p>Early genetic studies with mutants resistant to various purine analogs had indicated that <em>Aspergillus nidulans</em> contains two major transport systems for uptake of purine bases, the one for uric acid and xanthine (identified later as UapA) and the other for hypoxanthine, adenine and guanine (identified later as AzgA); the AzgA system had been hypothesized on the basis of the isolation of mutants resistant to (the toxicity effect of) 8-<u>az</u>a<u>g</u>uanine (<em>azg</em> mutants) (Darlington and Scazzocchio, 1967); the <em>azgA</em> gene was later cloned using transposon mutagenesis and shown to transport hypoxanthine and recognize adenine and guanine in competition assays (Cecchetto et al., 2004); in mycelia, <em>azgA</em> is, like other genes of purine catabolism, induced by uric acid and repressed by ammonium (Cecchetto et al., 2004)</p>

    <p>AzgA displays high affinity for hypoxanthine (<em>K</em><sub>m</sub> 1.5 μM), adenine (<em>K</em><sub>i</sub> 3 μM) and guanine (<em>K</em><sub>i</sub> 3 μM) and several related purine analogs including 2,6-diaminopurine (<em>K</em><sub>i</sub> 2 μM), 6-mercaptopurine, 6-chloropurine, 6-thioguanine (<em>K</em><sub>i</sub> in the range 60-80 μM), purine (<em>K</em><sub>i</sub> 99 μM), 8-azaguanine (<em>K</em><sub>i</sub> 133 μM), 8-azaxanthine (<em>K</em><sub>i</sub> 11 μM); it does not bind or transport oxidized purines (xanthine, uric acid) or pyrimidines (Goudela et al., 2006); theoretical models of AzgA were constructed on the template of UraA, and substrate docking studies and molecular dynamics were performed for AzgA in complex with hypoxanthine, guanine or adenine (Krypotou et al., 2014)</p>

    <p>&nbsp;</p>

    <h4>AfAzgA of Aspergillus fumigatus: Adenine/guanine/hypoxanthine permease</h4>

    <h4>UniProt_ A0A229Y8L1</h4>

    <p>Proton-gradient dependent hypoxanthine (and adenine and guanine) transporter with properties similar to AzgA of <em>Aspergillus nidulans</em>; it displays high affinity for hypoxanthine (<em>K</em><sub>m</sub> 0.8 μM in <em>A. fumigatus</em>, or 5.7 μM when heterologously expressed in <em>A. nidulans</em>), adenine (<em>K</em><sub>i</sub> 1.0 μM in <em>A. fumigatus</em>, or 3.5 μM when expressed in <em>A. nidulans</em>) and guanine (<em>K</em><sub>i</sub> 0.9 μM in <em>A. fumigatus</em>, or 7.6 μM when expressed in <em>A. nidulans</em>) and several related purine analogs including 2,6-diaminopurine (<em>K</em><sub>i</sub> 3 μM), purine, 6-mercaptopurine, 6-chloropurine, 6-thioguanine (<em>K</em><sub>i</sub> in the range 30-50 μM) and 8-azaguanine (<em>K</em><sub>i</sub> 130 μM) (Goudela et al., 2008)</p>

    <p>&nbsp;</p>

    <h4>AtAzg1of Arabidopsis thaliana: Adenine/guanine permease</h4>

    <h4>UniProt_Q9SRK7</h4>

    <p>An <em>AtAzg1</em>-deficient homozygous mutant line of <em>Arabidopsis thaliana</em> (<em>AtAzg1-1</em>) is less sensitive to 8-azaguanine or 8-azaadenine and shows significant reduction in the uptake of [<sup>3</sup>H]guanine and [<sup>3</sup>H]adenine relative to wild type; the double mutant <em>AtAzg1-1/AtAzg2-1</em> shows much more clear decrease in sensitivity to 8-azaguanine and 8-azaadenine and reduction in the uptake of [<sup>3</sup>H]guanine and [<sup>3</sup>H]adenine; heterologously expressed AtAzg1 increases the sensitivity to 0.1 mM 8-azaguanine and 2 mM 8-azaadenine (but not 5-FC (5-fluorocytosine)) and the uptake of [<sup>3</sup>H]guanine and [<sup>3</sup>H]adenine in an Fcy2-deficient <em>S. cerevisiae</em> strain (RG191) (Mansfield et al., 2009)</p>

    <p>&nbsp;</p>

    <h4>AtAzg2 of Arabidopsis thaliana: Adenine/guanine permease</h4>

    <h4>UniProt_Q84MA8</h4>

    <p>An <em>AtAzg1</em>-deficient homozygous mutant line of <em>Arabidopsis thaliana</em> (<em>AtAzg2-1</em>) is less sensitive to 8-azaguanine or 8-azaadenine than wild type; it also shows some resistance to 5-FU; the double mutant <em>AtAzg1-1/AtAzg2-1</em> shows much more clear decrease in sensitivity to 8-azaguanine and 8-azaadenine and reduction in the uptake of [<sup>3</sup>H]guanine and [<sup>3</sup>H]adenine; heterologously expressed AtAzg2 increases the sensitivity to 0.1 mM 8-azaguanine and 2 mM 8-azaadenine (but not 5-FC) and the uptake of [<sup>3</sup>H]guanine and [<sup>3</sup>H]adenine in an Fcy2-deficient <em>S. cerevisiae</em> strain (RG191) (Mansfield et al., 2009)</p>

    <p>&nbsp;</p>

    <h4>PhZ of Phanerochaete chrysosporium: adenine, hypoxanthine and guanine transporter</h4>

    <p>PhZ is an adenine, hypoxanthine, and guanine transporter with properties similar to AzgA (based on growth tests on purines and other nitrogen sources and protection against the toxicity of 8-azaguanine in growth tests combining 8-azaguanine with other purines as putative competitors); adenine and hypoxanthine were detected as substrates in direct growth assays, whereas guanine was detected indirectly, from its competition effect on 8-azaguanine allowing protection from the 8-azaguanine toxicity (Barrago-Vega et al., 2017)</p>

    <p>&nbsp;</p>

</div>
