<div class="col-8 offset-2">
    <h2><strong>References</strong></h2>

    <p>&nbsp;</p>

    <ol>
        <li>Alexander, C. R., Dingman, D. W., Schultes, N. P., and Mourad, G. S. (2018). The solute transport profile of two Aza-guanine transporters from the honey bee pathogen <em>Paenibacillus larvae</em>. <em>FEMS Microbiol. Lett.</em> <strong>365</strong>, in press.</li>
        <li>Alguel, Y., Amillis, S., Leung, J., Lambrinidis, G., Capaldi, S., Scull, N.J., Craven, G., Iwata, S., Armstrong, A., Mikros, E., Diallinas, G., Cameron, A. D., and Byrne, B. (2016). Structure of eukaryotic purine/H<sup>+</sup> symporter UapA suggests a role for homodimerization in transport activity. <em>Nature Commun.</em> <strong>7</strong>: 11336.</li>
        <li>Amillis, S. Koukaki, M., and Diallinas, G. (2001). Substitution F569S converts UapA, a specific uric acid-xanthine transporter, into a broad specificity transporter for purine-related solutes<em>. J. Mol. Biol.</em> <strong>313</strong>, 765-774.</li>
        <li>Amillis, S., Kosti, V., Pantazopoulou, A., Mikros, E., and Diallinas, G. (2011). Mutational analysis and modeling reveal functionally critical residues in transmembrane segments 1 and 3 of the UapA transporter. <em>J. Mol. Biol.</em> <strong>411</strong>, 567-580.</li>
        <li>Andersen, P. S., Frees, D., Fast, R., and Mygind, B. (1995). Uracil uptake in <em>Escherichia coli</em> K-12: isolation of UraA mutants and cloning of the gene.<em> J. Bacteriol. </em><strong>177</strong>, 2008-2013.</li>
        <li>Arsène-Ploetze, F.,&nbsp;Nicoloff,&nbsp;H., Kammerer, B.,&nbsp;Martinussen,&nbsp;J., and Bringel, F. (2006). Uracil salvage pathway in <em>Lactobacillus plantarum</em>: transcription and genetic studies. <em>J. Bacteriol.</em> 188, 4777-4786.</li>
        <li>Barrago-Vega, M., Romero, H., Richero, M., Cerdeiras, M. P., and Cecchetto, G. (2017). Functional characterization of two novel purine transporters from the Basidiomycota <em>Phanerochaete chrysosporium</em>. <em>Gene </em>601, 1–10.</li>
        <li><cite>Batey, R. T., Gilbert, S. D., and Montagne, R. K. (2004). Structure of a natural guanine-responsive riboswitch complexed with the metabolite hypoxanthine. </cite><cite>Nature</cite><cite> 432, 411-415.</cite></li>
        <li>Botou, M., Lazou, P., Papakostas, K., Lambrinidis, G., Evangelidis, T., Mikros. E., and Frillingos, S. (2018). Insight on specificity of uracil permeases of the NAT/NCS2 family from analysis of the transporter encoded in the pyrimidine utilization operon of <em>Escherichia coli</em>. <em>Mol. Microbiol. </em><strong>108</strong>, 204-219.</li>
        <li>Burton K. (1983). Transport of nucleic acid bases into<em> Escherichia coli. J. Gen. Microbiol. </em><strong>129</strong>, 3505–3513.</li>
        <li>Burton K. (1994). Adenine transport in <em>Escherichia coli</em>. <em>Proc. Biol. Sci.</em> <strong>255</strong>, 153–157</li>
        <li>Cecchetto G., Amillis S., Diallinas G., Scazzocchio C., and Drevet C. (2004). The AzgA purine transporter of <em>Aspergillus nidulans</em>: characterization of a protein belonging to a new phylogenetic cluster. <em>J. Biol. Chem.</em> <strong>279</strong>, 3132–3141</li>
        <li>Cho, B. K., Federowicz, S. A., Embree, M., Park, Y. S., Kim, D., and Palsson, B. Ø. (2011). The PurR regulon in <em>Escherichia coli</em> K-12 MG1655. <em>Nucleic Acids Res.</em> <strong>39</strong>, 6456-6464.&nbsp;</li>
        <li>Christiansen, L. C.,<cite> Schou, S., M., Nygaard, P., and </cite>Saxild, H. H. <cite>(</cite>1997<cite>).</cite> Xanthine metabolism in <em>Bacillus subtilis</em>: characterization of the <em>xpt-pbuX</em> operon and evidence for purine- and nitrogen-controlled expression of genes involved ion xanthine salvage and catabolism<cite>. </cite><em>J</em><em>. Bacteriol.</em> <strong>179</strong><cite>,</cite> 2540<cite>–</cite>2550<cite>.</cite></li>
        <li>Darlington, A. J., and Scazzocchio, C. (1967). Using analogues and the substrate selectivity of mutants in analysis of purine uptake and breakdown in <em>Aspergillus nidulans</em>. <em>J. Bacteriol.</em> <strong>93</strong>, 937-940.</li>
        <li>Diallinas, G. (2014). Understanding transporter specificity and the discrete appearance of channel-like gating domains in transporters. <em>Front. Pharmacol.</em> <strong>5</strong>:207.</li>
        <li>Diallinas, G., and Scazzocchio, C. (1989). A gene coding for the uric acid-xanthine permease of <em>Aspergillus nidulans</em>: inactivation cloning, characterization and sequence of a cis-acting mutation. <em>Genetics</em> <strong>122</strong>, 341-350.</li>
        <li>Diallinas, G., Gorfinkiel, L., Arst, H. N. Jr., Sophianopoulou, V., Cecchetto, G., and Scazzocchio, C. (1995). Genetic and molecular characterization of a gene encoding a wide specificity purine permease of <em>Aspergillus nidulans</em> reveals a novel family of transporters conserved in prokaryotes and eukaryotes. <em>J. Biol. Chem. </em>&nbsp;<strong>270</strong>, 8610-8622.</li>
        <li>Diallinas, G., Valdez, J., Sophianopoulou, V., Rosa, A., and Scazzocchio, C. (1998). Chimeric purine transporters of <em>Aspergillus nidulans</em> define a domain critical for function and specificity conserved in bacterial, plant and metazoan homologues. <em>EMBO J.</em> <strong>17</strong>, 3827-3837.</li>
        <li>Evangelinos, M., Martzoukou, O., Chorozian, K., Amillis, S., and Diallinas, S. (2016). BsdA<sup>Bsd2</sup>-dependent vacuolar turnover of a misfolded version of UapA transporter along the secretory pathway: prominent role of selective autophagy. <em>Mol. Microbiol.</em> <strong>100</strong>, 893-911.&nbsp;&nbsp;</li>
        <li><cite>Frillingos, S. (2012). Insights to the evolution of Nucleobase-Ascorbate Transporters (NAT/NCS2 family) from the Cys-scanning analysis of xanthine permease XanQ. </cite><cite>Int. J. Biochem. Mol. Biol.</cite><cite> <strong>3</strong>, 250-272. </cite>&nbsp;</li>
        <li>Genee, H. J.<cite>, Bali, A. P., Petersen, S. D., Siedler, S., Bonde, M. T., Gronenberg, L. S., Kristensen, M., Harrison, S. J., and </cite>Sommer, M. H. <cite>(</cite>2016<cite>).</cite> Functional mining of transporters using synthetic selections<cite>. </cite><em>Nature Chem</em><em>. Biol.</em> <strong>12</strong><cite>,</cite> 1015<cite>–</cite>1022<cite>.</cite></li>
        <li>Georgopoulou, E., Mermelekas, G., Karena, E., and Frillingos, S. (2010). Purine substrate recognition by the nucleobase-ascorbate transporter signature motif in the YgfO xanthine permease: Asn-325 binds and Ala-323 senses substrate. <em>J. Biol. Chem.</em> <strong>285</strong>, 19422-19433.</li>
        <li>Gorfinkiel, L., Diallinas, G., and Scazzocchio, C. (1993). Sequence and regulation of the <em>uapA</em> gene encoding a uric acid-xanthine permease in the fungus <em>Aspergillus nidulans</em>. <em>J. Biol. Chem.</em><strong>122</strong>, 341-350.</li>
        <li>Goudela, S., Karatza, P., Koukaki, M., Frillingos, S., and Diallinas, G. (2005). Comparative substrate recognition by bacterial and fungal purine transporters of the NAT/NCS2 family. <em>Mol. Membr. Biol.</em> <strong>23</strong>, 263–275.</li>
        <li>Goudela, S., Reichard, U., Amillis, S., and Diallinas, G. (2008). Characterization and kinetics of the major purine transporters in <em>Aspergillus fumigatus</em>. <em>Fung. Genet. Biol. </em><strong>45</strong>, 459-472.</li>
        <li>Goudela, S., Tsilivi, H., and Diallinas, G. (2006). Comparative kinetic analysis of AzgA and Fcy21p, prototypes of the two major fungal hypoxanthine-adenine-guanine transporter families. <em>Mol. Membr. Biol.</em> <strong>23</strong>, 291–303.</li>
        <li>Gournas, C., Amillis, S., Vlanti, A., and Diallinas, G. (2010). Transport-dependent endocytosis and turnover of a uric acid-xanthine permease. <em>Mol. Microbiol.</em> <strong>75</strong>, 246-260.&nbsp;</li>
        <li>Jain, S., Showman A. C., and Jewett, M. W. (2015). Molecular dissection of a <em>Borrelia burgdorferi</em> in vivo essential purine transport system. <em>Infect</em><em>.</em><em> Immun</em><em>.</em> <strong>83</strong>, 2224-2233.</li>
        <li>Jain, S., Sutchu, S., Rosa, P. A., Byram, R., and Jewett, M. W. (2012). <em>Borrelia burgdorferi</em> harbors a transport system essential for purine salvage and mammalian infection. <em>Infect</em><em>.</em><em> Immun</em><em>.</em> <strong>80</strong>, 3086-3093.</li>
        <li>Jewett, M. W., Bestor, A., Byram, R., Gherardini, F., and Rosa, P. A. (2009). GuaA and GuaB are essential for survival in the tick-mouse infection cycle. <em>J. Bacteriol.</em> <strong>191</strong>, 6231-6241.</li>
        <li>Jewett, M. W., Byram, R., Bestor, A., Tilly, K., Lawrence, K., Burtnick, M. N., Gherardini, F., and Rosa, P. A. (2007). Genetic basis for retention of a critical virulence plasmid of <em>Borrelia burgdorferi</em>. <em>Mol. Microbiol.</em> <strong>66</strong>, 975-990.</li>
        <li>Johansen, L. E., Nygaard, P., Laasen, C., Agersø, Y., and Saxild, H. H. (2003). Definition of a second <em>Bacillus subtilis</em> <em>pur</em> regulon comprising the<em> pur</em> and <em>xpt-pbuX</em> operons plus<em> pbuG</em>, <em>nupG</em> (<em>yxjA</em>), and <em>pbuE</em> (<em>ydhL</em>). <em>J. Bacteriol.</em> <strong>185</strong>, 5200-5209.</li>
        <li>Karatza, P., and Frillingos, S. (2005). Cloning and functional characterization of two bacterial members of the NAT/NCS2 family in <em>Escherichia coli</em>. <em>Mol. Membr. Biol.</em> <strong>22</strong>, 251-261.</li>
        <li>Karatza, P.<cite>,</cite> Panos, P.<cite>, </cite>Georgopoulou, E.<cite>, and</cite> Frillingos, S. <cite>(</cite>2006<cite>).</cite> Cysteine-scanning analysis of the nucleobase-ascorbate transporter signature motif in YgfO permease of <em>Escherichia coli</em>: Gln-324 and Asn-325 are essential and Ile-329–Val-339 form an alpha-helix<cite>. </cite><em>J. Biol. Chem.</em> <strong>281</strong><cite>,</cite> 39881<cite>–</cite>39890<cite>.</cite></li>
        <li>Karena, E., and Frillingos, S. (2009). Role of intramembrane polar residues in the YgfO xanthine permease: His31 and Asn93 are crucial for affinity and specificity and Asp304 and Glu272 are irreplaceable. <em>J. Biol. Chem.</em> <strong>284</strong><cite>,</cite> 24257<cite>–</cite>24268.</li>
        <li>Karena, E., and Frillingos, S. (2011). The role of transmembrane segment TM3 in the xanthine permease XanQ of <em>Escherichia coli</em>.<em> J. Biol. Chem.</em> <strong>286</strong><cite>,</cite> 39595<cite>–</cite>39605<cite>.</cite></li>
        <li>Karena, E., Tatsaki, E., Lambrinidis, G., Mikros, E., and Frillingos, S. (2015). Analysis of conserved NCS2 motifs in the <em>Escherichia coli</em> xanthine permease XanQ. <em>Mol. Microbiol.</em> <strong>98</strong>, 502-517.</li>
        <li>Kim, K.-S., Pelton, G. J., Inwood, W. B., Anderesen, U., Kustu, S., and Wemmer, D. E. (2010).The Rut pathway for pyrimidine degradation: novel chemistry and toxicity problems. <em>J. Bacteriol.</em> <strong>192</strong>, 4089-4102.</li>
        <li>Kosti, V., Lambrinidis, G., Myrianthopoulos, V., Diallinas, G., and Mikros, E. (2012). Identification of the substrate recognition and transport pathway in a eukaryotic member of the nucleobase-ascorbate transporter (NAT) family. <em>PLOS One</em> <strong>7</strong>: e41939.</li>
        <li>Kosti, V., Papageorgiou, I., and Diallinas, G. (2010). Dynamic elements at both cytoplasmically and extracellularly facing sides of the UapA transporter selectively control the accessibility of substrates to their translocation pathway. <em>J. Mol. Biol.</em> <strong>397</strong>, 1132-1143.</li>
        <li>Koukaki, M., Vlanti, A., Goudela, S., Pantazopoulou, A., Gioule, H., Tournaviti, S., and Diallinas, G. (2005). The nucleobase-ascorbate transporter (NAT) signature motif in UapA defines the function of the purine translocation pathway. <em>J. Mol. Biol.</em> <strong>350</strong>, 499-513.</li>
        <li>Kozmin S. G., Stepchenkova E. I., Chow S. C., and Schaaper R. M. (2013). A critical role for the putative NCS2 nucleobase permease YjcD in the sensitivity of <em>Escherichia coli</em> to cytotoxic and mutagenic purine analogs. <em>mBio </em><strong>4</strong>, e00661–13</li>
        <li>Krypotou, A., Lambrinidis, G., Evangelidis, T., Mikros, E., and Diallinas, G. (2014). Modelling, substrate docking and mutational analysis identify residues essential for function and specificity of the major fungal purine transporter AzgA. <em>Mol. Microbiol.</em> <strong>93</strong>, 129-145.</li>
        <li>Krypotou, E., and Diallinas, G. (2014). Transport assays in filamentous fungi: kinetic characterization of the UapC purine transporter of <em>Aspergillus nidulans</em>. <em>Fungal Gen. Biol.</em> <strong>63</strong>, 1-8.&nbsp;&nbsp;&nbsp;</li>
        <li>Krypotou, E., Scazzocchio, C., and Diallinas, G. (2016). Functional characterization of NAT/NCS2 proteins of <em>Aspergillus brasiliensis </em>reveals a genuine xanthine-uric acid transporter and an intrinsically misfolded polypeptide. <em>Fungal Gen. Biol.</em> <strong>75</strong>, 56-63.&nbsp;&nbsp;&nbsp;</li>
        <li>Leung, J., Cameron, A. D., Diallinas, G., and Byrne, B. (2013). Stabilizing the heterologously expressed uric acid-xanthine transporter from the lower eukaryote <em>Aspergillus nidulans</em>. <em>Mol. Membr. Biol.</em> <strong>30</strong>, 32-42.</li>
        <li>Loh, K. D., Gyaneshwar, P., Markenscoff Papadimitriou, E., Fong, R., Kim, K.-S., Parales, R., Zhou, Z., Inwood, W., and Kustu, S. (2006). A previously undescribed pathway for pyrimidine catabolism.&nbsp; <em>Proc. Natl. Acad. Sci. USA</em> <strong>103</strong>, 5114-5119.</li>
        <li>Lu, F., Li, S., Jiang, Y., Jiang, J., Fan, H., Lu, G., Deng, D., Dang, S., Zhang, X., Wang, J., and Yan, N. (2011). Structure and mechanism of the uracil transporter UraA. <em>Nature </em><strong>472</strong>, 243-246.</li>
        <li>Mandal, M., Boese, B., Barrick, J. E., Winkler, W. C., and Breaker, R. R. (2003). Riboswitches control fundamental biochemical pathways in <em>Bacillus subtilis</em> and other bacteria. <em>Cell</em> <strong>113</strong>, 577-586.</li>
        <li>Mansfield T. A., Schultes N. P., and Mourad G. S. (2009) AtAzgA1 and AtAzgA2 comprise a novel family of purine transporters in <em>Arabidopsis</em>. <em>FEBS Lett.</em> <strong>583</strong>, 481–486.</li>
        <li>Marbach, D., Costello, J. C., Kuffner, R., Vega, N. M., Prill, R. J., Camacho, D. M., Allison, K. R., DREAM5 Consortium, Kellis, M., Collins, J. J., and Stolovitzky, G. (2012) Wisdom of crowds for robust gene network inference. <em>Nature Methods</em> <strong>9</strong>, 796-804.&nbsp;</li>
        <li>Martzoukou, O., Karachaliou, M., Yalelis, V., Leung, J., Byrne, B., Amillis, S., and Diallinas, G. (2015). Oligomerization of the UapA purine transporter is critical for ER-exit, plasma membrane localization and turnover.<em> J. Mol. Biol.</em> <strong>427</strong>, 2679-2696.</li>
        <li>Meintanis, C., Karagouni, A. D., and Diallinas, G. (2000). Amino acid residues N<sup>450</sup> and Q<sup>449</sup> are critical for the uptake capacity and specificity of UapA, a prototype of a universally conserved nucleobase-ascorbate transporter family. <em>Mol. Membr. Biol.</em> <strong>17</strong>, 47-58.</li>
        <li>Mermelekas, G., Georgopoulou, E., Kallis, A., Botou, M., Vlantos, V., and Frillingos, S. (2010). Cysteine-scanning analysis of helices TM8, TM9a, and TM9b and intervening loops in the YgfO xanthine permease: a carboxyl group is essential at Asp-276. <em>J. Biol. Chem.</em> <strong>285</strong>, 35011-35020.</li>
        <li>Pantazopoulou, A., and Diallinas, G. (2006). The first transmembrane segment (TMS1) of UapA contains determinants necessary for expression in the plasma membrane and purine transport. <em>Mol. Membr. Biol.</em> <strong>23</strong>, 337-348.</li>
        <li>Papageorgiou, I., Gournas, C., and Diallinas, G. (2008). Specific interdomain synergy in the UapA transporter determines its unique specificity for uric acid among NAT carriers. <em>J. Mol. Biol.</em> <strong>382</strong>, 1121-1135.&nbsp;</li>
        <li>Papakostas, K.<cite>, and </cite>Frillingos, S. <cite>(</cite>2012<cite>).</cite> Substrate selectivity of YgfU, a uric acid transporter from <em>Escherichia coli</em><cite>. </cite><em>J. Biol. Chem</em><em>.</em> <strong>287</strong><cite>,</cite> 15684<cite>–</cite>15695<cite>.</cite></li>
        <li>Papakostas, K., Botou, M., and Frillingos, S. (2013). Functional identification of the hypoxanthine/guanine transporters YjcD and YgfQ and the adenine transporters PurP and YicO of <em>Escherichia coli </em>K-12. <em>J. Biol. Chem.</em> <strong>288</strong>, 36827-36840.</li>
        <li>Papakostas, K.<cite>, Georgopoulou, E., and </cite>Frillingos, S. <cite>(</cite>2008<cite>).</cite> Cysteine-scanning analysis of putative helix XII in the YgfO xanthine permease: Ile-432 and Asn-430 are important<cite>. </cite><em>J. Biol. Chem</em><em>.</em> <strong>283</strong><cite>,</cite> 13666<cite>–</cite>13678<cite>.</cite></li>
        <li>Roy-Burman, S., and Visser, D. W. (1975). Transport of purines and deoxyadenosine in <em>Escherichia coli</em>. <em>J. Biol. Chem. </em><strong>250</strong>, 9270-9275.</li>
        <li>Saxild, H. H., and Nygaard, P. (1987). Genetic and physiological characterization of <em>Bacillus subtilis</em> mutants resistant to purine analogs. <em>J. Bacteriol.</em> <strong>169</strong>, 2977–2983.</li>
        <li>Saxild, H. H., Brunstedt, K., Nielsen, K. I., Jarmer, H., and Nygaard, P. (2001). Definition of the <em>Bacillus subtilis</em> PurR operator using genetic and bioinformatic tools and expansion of the PurR regulon with <em>glyA</em>, <em>guaC</em>, <em>pbuG</em>, <em>xpt-pbuX</em>, <em>yqhZ-folD</em>, and <em>pbuO</em>. <em>J. Bacteriol.</em> <strong>183</strong>, 6175-6183.</li>
        <li>Schultz, A. C., <cite>Nygaard, P., and </cite>Saxild, H. H. <cite>(</cite>2001<cite>).</cite> Functional analysis of 14 genes that constitute the purine catabolic pathway in <em>Bacillus subtilis</em> and evidence for a novel regulon controlled by the PucR transcription activator. <em>J. Bacteriol.</em> <strong>183</strong>, 3293-3302.</li>
        <li>Turner, R. J., Lu, Y. and Switzer, R. L. (1994). Regulation of the <em>Bacillus subtilis</em> pyrimidine biosynthetic (<em>pyr</em>) gene cluster by an autogenous transcriptional attenuation mechanism. <em>J. Bacteriol.</em> 176, 3708-3722.</li>
        <li>Vlanti, A., Amillis, S., Koukaki, M., and Diallinas, G. (2006). A novel-type substrate-selectivity filter and ER-exit determinants in the UapA purine transporter. <em>J. Mol. Biol.</em> <strong>357</strong>, 808-819.</li>
        <li>Xi, H., Schneider, B. L., and Reitzer, L. (2000). Purine catabolism in <em>Escherichia coli</em> and function of xanthine dehydrogenase in purine salvage. <em>J. Bacteriol.</em> <strong>182</strong>, 5332-5341.</li>
        <li>Yu, X., Yang, G., Yan, C., Baylon, J. L., Jiang, J., Fan, H., Lu, G., Hasegawa, K., Okumura, H., Wang, T., Tajkhorshid, E., Li, S., and Yan. N. (2017). Dimeric structure of the uracil:proton symporter UraA provides mechanistic insights into the SLC4/23/26 transporters. <em>Cell Res.</em> <strong>27</strong>, 1020-1033.</li>
        <li>Zimmer, D.P., Soupene, E., Lee, H.L., Wendisch, V.F., Khodursky, A.B., Peter, B. J., Bender, R.A., and Kustu, S. (2000). Nitrogen regulatory protein C-controlled genes of <em>Escherichia coli</em>: Scavenging as a defense against nitrogen limitation. <em>Proc. Natl. </em><em>Acad. Sci. USA </em><strong>97</strong>, 14674-14679.</li>
    </ol>

    <p>&nbsp;</p>
    <h2><strong>Supplementary Data</strong></h2>

    <p>&nbsp;</p>
    <p>You can download supplementary data <a href="http://localhost:8080/images/Supplementary_data.zip" download>here</a></p>

</div>
