<div class="col-8 offset-2">
    <img src="/images/intro.png" style="width:100%"/>
    <p style="text-align: center; font-size: small">Source: Karena & Frillingos, J Biol Chem. 2011 Nov 11;286(45):39595-605.</p>
    <h2>Introduction</h2>
    <p>Nucleobase transporters are members of the APC superfamily of secondary active transporters (Saier et al, 2016) and are very important for the cells since their substrate uptake may function as nitrogen/carbon source, affect energy production, signaling, replication, transcription and protein synthesis through the salvage pathways for nucleotide synthesis (Papakostas et al, 2013; King et al, 2006). In addition to their important direct role on the central metabolism of the cell, they have been regarded as potentially important drug targets for viral/parasitic diseases, for cancer (Köse & Schiedel, 2009) or even mammalian physiology (sleep, drug and alcohol abuse, pain), since they regulate the levels of free nucleosides in circulation .</p>
    <p>There exist 4 known families of transporters that use nucleobases as their main substrates, the Nucleobase-Cation Symporter (NCS) families NCS1 (absent in Metazoa) and NCS2 (or NAT/NCS2) (universally present) that belong to the APC superfamily, and two minor families that are found exclusively in Plants and belong to the distinct DMT superfamily. The NAT/NCS2 family distinguishes from the others because it is very well conserved and wide-spread across many different taxa (Frillingos, 2012).</p>
    <p>Based on an evolutionary and bioinformatics analysis pipeline that we have already implemented in another superfamily, the tRNA synthetases – see (Chaliotis et al. 2017), we have generated a hierarchical evolutionary classification of the NAT/NCS2 family for prokaryotes. This classification helps to assign family members to evolutionary groups with distinct substrate preferences and identify key conserved and functional amino acids that affect substrate specificity.</p>
    <p>Towards this end, we have generated Hidden Markov Models (HMMs) and conserved MEME motifs for each level and evolutionary group. These HMMs and MEME motifs are used in our webserver to scan, identify and classify NAT/NCS2 proteins. In addition, structural and functional information from mutational studies is integrated to easily identify key amino acids and assess their level of conservation.</p>
    <br />
    <h2>Evolutionary Classification</h2>
    <img src="/images/intro1.png" style="width:100%"/>
    <div class="caption" style="text-align: center">Hierarchical evolutionary classification of prokaryotic NAT/NCS2 proteins into two subfamilies. Due to its large intragroup divergence, subfamily 1 is further divided into 6 major evolutionary clusters, based on phylogeny.</div>
    <br />
    <img src="/images/intro2.png" style="width:100%"/>
    <div class="caption" style="text-align: center">NAT/NCS2 subfamily 1 phylogeny and designation of 6 major evolutionary clusters.</div>
    <br />
    <h2>Bibliography</h2>
    <ul>
        <li>Chaliotis, A., P. Vlastaridis, D. Mossialos, M. Ibba, H. D. Becker et al., 2017 The complex evolutionary history of aminoacyl-tRNA synthetases. Nucleic Acids Res. 45: 1059–1068.</li>
        <li>Frillingos, S., 2012 Insights to the evolution of Nucleobase-Ascorbate Transporters (NAT/NCS2 family) from the Cys-scanning analysis of xanthine permease XanQ. Int J Biochem Mol Biol 3: 250–272.</li>
        <li>King, A. E., M. A. Ackley, C. E. Cass, J. D. Young, and S. A. Baldwin, 2006 Nucleoside transporters: from scavengers to novel therapeutic targets. Trends Pharmacol. Sci. 27: 416–425.</li>
        <li>Köse, M., and A. C. Schiedel, 2009 Nucleoside/nucleobase transporters: drug targets of the future? Future Med Chem 1: 303–326.</li>
        <li>Papakostas, K., M. Botou, and S. Frillingos, 2013 Functional identification of the hypoxanthine/guanine transporters YjcD and YgfQ and the adenine transporters PurP and YicO of Escherichia coli K-12. J. Biol. Chem. 288: 36827–36840.</li>
        <li>Saier, M. H., V. S. Reddy, B. V. Tsu, M. S. Ahmed, C. Li et al., 2016 The Transporter Classification Database (TCDB): recent advances. Nucleic Acids Res. 44: D372-379.</li>
    </ul>

</div>
